src/snippets/uncurated-ncbi-dataset.rst

```diff
@@ -0,0 +1,37 @@
+1. Enter an interactive Nextstrain shell to be able to run the NCBI Datasets CLI commands without installing them separately.
```

Comment:
There are many ways to get to the some outcome and docs like these always have to balance covering all the ways to get there vs detailing just one. (Examples are everywhere - we detail nextstrain CLI rather than the snakemake commands it's actually running, we assume non-ambient runtimes.) Sometimes adding little pointers to indicate the links between the different methods would be really helpful. In this example, we are running commands which are largely recreating the steps we just ran in the "running an ingest workflow" section - steps 2+3 are identical to `snakemake --cores 1 --notemp fetch_ncbi_dataset_package`.

I'd change the introduction slightly to indicate this:

```diff
- If you want to see the uncurated NCBI Datasets data to decide what changes
- you would like to make to the workflow, you can run the following:
+ If you want to see the uncurated NCBI Datasets data to decide what changes
+ you would like to make to the workflow, you can download the raw NCBI data
+ by manually running commands very similar to those the pipeline used earlier
+ when "running an ingest workflow"
```

I'd be interested in adopting a consistent approach throughout the docs where we add [little hints](https://docutils.sourceforge.io/docs/ref/rst/directives.html#hint) to explain links between what we're detailing and the other ways it can be / is done. Using this as an example (and based off the text at the end of this snippet), I'm imagining a hint section after the commands such as:

> These commands are actually run by the ingest pipeline with some minor differences. The ingest pipeline restricts the columns to those defined in `config["ncbi_datasets_fields"]`, modifies the header names using `config["curate"]["field_map"]` (which maps the [human-readable strings NCBI uses](https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/dataformat/tsv/dataformat_tsv_virus-genome/#fields) to ones more typical in Nextstrain pipelines), and changes the actual data via a number of curation steps.

I can add similar comments throughout these docs if you are interested in adopting this style, but wanted to discuss here first.

Reply:
Yeah, I think there's a range of views out there (including in our group). 

>  I think including multiple/all ways to achieve the task can be distracting and overwhelming for a new user. 

I'm not asking for documenting all of the ways, just adding little hints so that people can join the dots between concepts that may appear separate / independent. In this case it's providing clarifying hints that this section is very related to the section they've just run, even though on the surface they seem completely different.

Reply:
Added your suggested hints in https://github.com/nextstrain/docs.nextstrain.org/pull/195/commits/6019d504aaa114e8c928c27faa3c676fd6a71d18. Feel free to add more!